eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/InMemoryLeaseManager.java

```diff
@@ -297,7 +298,7 @@ public void notifyOnSteal(String expectedOwner, String partitionId, Callable<?>
 
     private static class InMemoryLeaseStore {
         static final InMemoryLeaseStore SINGLETON = new InMemoryLeaseStore();
-        private static int leaseDurationInMilliseconds;
+        private volatile int leaseDurationInMilliseconds;
```

Comment:
is this the correct fix? The original code suggests that there should be a single instance of this across multiple instances of InMemoryLeaseStore. Did you mean `private static volatile`?

Reply:
I think this change is OK. There is only ever one instance of InMemoryLeaseStore, the SINGLETON, which is shared across all instances of InMemoryLeaseManager. I can't think of a reason for this member to be static.